configs/components/cpp-pcp-client.rb

```diff
@@ -19,6 +23,25 @@
   elsif platform.is_solaris?
     cmake = "/opt/pl-build-tools/i386-pc-solaris2.#{platform.os_version}/bin/cmake"
     toolchain = "-DCMAKE_TOOLCHAIN_FILE=/opt/pl-build-tools/#{settings[:platform_triple]}/pl-build-toolchain.cmake"
+  elsif platform.is_windows?
+    pkg.build_requires "cmake"
+    pkg.build_requires "pl-toolchain-#{platform.architecture}"
+    pkg.build_requires "pl-boost-#{platform.architecture}"
+
+    arch = platform.architecture == "x64" ? "64" : "32"
+    make = "#{platform.drive_root}/tools/mingw#{arch}/bin/mingw32-make"
+    prefix = platform.convert_to_windows_path(settings[:prefix])
```

Comment:
We could create a new setting in the project file, `windows_prefix = platform.convert_to_windows_path(settings[:prefix])`, but we can't have vanagon try to create the directory or any directories that inherit from it. Is that what you're thinking? We still have to do the special prefix casing though